recipes/lemon/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "lemon" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "71b7c725f4c0b4a8ccb92eb87b208701586cf7a96156ebd821ca3ed855bad3c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - glpk_win.diff    # [win]
+
+build:
+  number: 0
+  features:
+    - vc9           # [win and py27]
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python        # [win]
+    - toolchain
+    - cmake
+    - glpk
+    - vc 9          # [win and py27]
+    - vc 14         # [win and py>=35]
+    - coincbc       # [unix]
+    - doxygen       # [unix]
```

Comment:
It looks like the windows build of coincbc is disabled:

https://github.com/conda-forge/coincbc-feedstock

(I presume the usual fortran issues on windows?)

I'll try to enable doxygen to have the documentation built and installed on windows as well.